### ISSUE TITLE
grommet check lint also JSX 💭 

### ISSUE THREAD

**Issue by leomoreno** [grommet/grommet-cli#34]
Hi, I start using grommet recently and came up with some weird situation.
The `test` command that under the hood runs `grommet check` is not linting my _.jsx_ files.

Digging a bit I found the situation is caused apparently by this [line on check command](https://github.com/grommet/grommet-cli/blob/master/src/commands/check.js#L42):

```javascript
eslintChild.send('**/*.js');
```

Am I missing something here? 🤔 
Is there a way to tell eslint to check also _.jsx_ files?

**Comment by alansouzati** [comment]
good point. sorry for taking such a long time to get back to you on this. are you up for a pull request to change this expression to also handle JSX extensions ?

```
eslintChild.send('**/*.jsx?');
```

I believe this will work :)

**Comment by elmorenocivitas** [comment]
Just let get back from holidays and I'll issue a PR :)

**Comment by alansouzati** [comment]
nice! thank you very much
